components/external_server/src/model/issue_tracker.py

```diff
@@ -83,6 +85,10 @@ class IssueTracker:
     issue_types_api = issue_creation_api + "/createmeta/%s/issuetypes"
     issue_browse_url = "%s/browse/%s"
     suggestions_api: str = "%s/rest/api/2/search?jql=summary~'%s~10' order by updated desc&fields=summary&maxResults=20"
+    epics_api: str = (
+        '%s/rest/api/2/search?jql=type=epic and ("Epic Status" != Done or "Epic Status" is empty) and '
+        "project=%s&fields=summary&maxResults=100"
```

Comment:
Didn't we have the max results for Jira defined as a constant somewhere?

Reply:
Yes, but that constant lives in the collector. I could move it to the data model, I'll check if that's feasible.

Reply:
We should really implement pagination properly. But that's too much work for now. I'll add an issue for that.